app/src/main/res/layout/history_list_item.xml

```diff
@@ -9,10 +9,24 @@
     android:importantForAccessibility="no"
     android:orientation="vertical">
 
+    <View
+        android:id="@+id/top_divider"
+        android:layout_width="match_parent"
+        android:layout_height="8dp" />
+
     <include
         android:id="@+id/recently_closed_nav_empty"
         layout="@layout/recently_closed_nav_item" />
 
+    <include
+        android:id="@+id/synced_history_nav_empty"
+        layout="@layout/synced_history_nav_item" />
+
+    <View
+        android:id="@+id/bottom_divider"
+        android:layout_width="match_parent"
+        android:layout_height="32dp" />
```

Comment:
It has always been confusing how we duplicate the content between `history_list_item` and `component_history` and toggle the visibility in different views based on the state when we should just have a clear distinction. Unfortunately, adding this here does have unwanted side effects because `history_list_item` is used in multiple places including Tab History and Recently Closed Tab.

So, this is what I am seeing in Recently closed tabs, which doesn't seem intended and should be addressed.

<img width="465" alt="Screen Shot 2022-04-13 at 4 08 02 PM" src="https://user-images.githubusercontent.com/1190888/163262354-c0764e16-efeb-4b55-b40c-4ad35d624447.png">

Reply:
Nice catch! Fixed.

Yes, it was also confusing to me. But it was already there, so I had to work with it.